crates/std_detect/src/detect/os/windows/aarch64.rs

```diff
@@ -0,0 +1,42 @@
+//! Run-time feature detection for Aarch64 on Windows.
+
+use crate::detect::{cache, Feature};
+
+/// Try to read the features from OS API.
+pub(crate) fn detect_features() -> cache::Initializer {
+    const PF_ARM_NEON_INSTRUCTIONS_AVAILABLE: u32 = 19;
+    const PF_ARM_V8_CRYPTO_INSTRUCTIONS_AVAILABLE: u32 = 30;
+    const PF_ARM_V8_CRC32_INSTRUCTIONS_AVAILABLE: u32 = 31;
+
+    extern "system" {
+        pub fn IsProcessorFeaturePresent(ProcessorFeature: u32) -> u32;
```

Comment:
the first `u32` is a `DWORD` and the second one is a `BOOL`, right ? Could we maybe use the same ctypes that win API uses for clarity? (e.g. just use a type alias `type DWORD = u32;` at the beginning of the file).